viewer/packages/cad-styling/src/AssetNodeCollection.ts

```diff
@@ -59,7 +60,10 @@ export class AssetNodeCollection extends NodeCollection {
       this._fetchResultHelper.interrupt();
     }
     const fetchResultHelper = new PopulateIndexSetFromPagedResponseHelper<AssetMapping3D>(
-      assetMappings => assetMappings.map(mapping => new NumericRange(mapping.treeIndex, mapping.subtreeSize)),
+      assetMappings =>
+        assetMappings
+          .filter(isCdfAssetMapping)
```

Comment:
Is this needed because of changed behaviour in v10, or is this a bug fix that is unrelated to the upgrade? If it is unrelated, please split it out in a separate PR.

Reply:
@larshagencognite This is needed because of certain typing updates in the SDK, yes